.github/workflows/macos-ci-x86_64.yaml

```diff
@@ -49,7 +49,9 @@ jobs:
           export SPACK_SYSTEM_CONFIG_PATH="${ENVDIR}/site"
 
           # Find external packages
-          spack external find --scope system --exclude bison --exclude openssl
+          spack external find --scope system --exclude bison --exclude openssl --exclude gettext
+          # Need to find gettext outside of default (presumed) system path for krb5
```

Comment:
```suggestion
          # Need to find gettext outside of default (presumed to be a system) path for krb5
```

Reply:
Should these changes (lines 53 and 54) be added to the NewSiteConfig documentation instructions for x86_64 based MacOS?